inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts

```diff
@@ -4,7 +4,7 @@ import { isProduction } from "./env.js"
 
 export function initErrorMonitoring() {
 	Sentry.init({
-		dsn: "https://paste-sentry-url.here",
+		dsn: "https://ba058608bbf997cfe6c5f8d093d31d5b@us.sentry.io/4506700407177216",
```

Comment:
@felixhaeberle is this deactiavated for dev mode? you don't want to spam sentry. also this should have been an env variable

Reply:
i copied the setup you did for the CLI, so I thought it was on purpose

Reply:
oh haha lol